src/Microsoft.AspNetCore.Blazor.Build/ReferenceFromSource.props

```diff
@@ -10,6 +10,10 @@
   
   <Import Project="$(MSBuildThisFileDirectory)targets\All.targets" />
 
+  <ItemGroup>
+    <PackageReference Include="Microsoft.AspNetCore.Razor.Design" Version="$(RazorPackageVersion)"/>
+  </ItemGroup>
```

Comment:
This package delivers a specific version of the razor toolset. Whichever is the 'root' package for Blazor client code should include this and also include its MSBuild assets.

Reply:
You need this **AND** the Razor SDK